### PR TITLE
Add JS classes

### PR DESCRIPTION
**Pull request by algerina**
In this PR I have used  JavaScript classes 
Create class methods to add and remove books.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable */
```
Comment (carlos-ssh) [inline index.js]: - Enable your linters and fix issues.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+const addBtn = document.querySelector('#addBtn');
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
```
Comment (carlos-ssh) [inline index.js:13]: - You need to have class methods to add and remove books. Check if add and espcially remove is working fine, eg, with duplicate title or author (if allowed), the right book is still removed.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @algerina , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (algerina) [reply index.js]: @car I have an issue with h ESLint, but I didn't know how to get rid of them. Can you give me some clues, please
coach
?

**Review by afanimartin — APPROVED** [review]
Hi @algerina, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**